frontend/src/bundles/common/components/avatar/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,31 @@
+import { Avatar as MUIAvatar, type AvatarProps } from '@mui/material';
+
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = AvatarProps & {
+    small?: boolean;
+};
+
+const Avatar: React.FC<Properties> = ({
+    small = false,
+    className,
+    ...props
+}) => {
+    return (
+        <MUIAvatar
+            className={getValidClassNames(
+                className,
+                styles.avatar,
+                small ? styles.small : '',
+            )}
+            {...props}
+        >
+            {/* default icon/text */}
```

Comment:
why do we need these comments?

Reply:
in case if we need to change default appearance of avatar and there will be no need to go mui docs. but I can delete it